Makefile

```diff
@@ -43,7 +44,7 @@ gofmt:
 
 .PHONY: bin
 bin:
-	go build  ${CFLAGS} -o $(EXECUTABLE) -ldflags=$(LD_FLAGS) cmd/kam/kam.go 
+	go build  ${CFLAGS} -o $(DIST_DIR)/$(EXECUTABLE) -ldflags=$(LD_FLAGS) cmd/kam/kam.go 
```

Comment:
AFAIK this is not a part of binary distribution. Its a regular binary creation on the platform you are executing, for example on linux platform it produce linux compiled bit and on windows it produce .exe file. I think it does not make sense to add this in distribution directory.

Reply:
what would you suggest?

Reply:
how about we split this into `make dist` and `make bin` ?